homeassistant/components/switch/fritzdect.py

```diff
@@ -86,17 +89,20 @@ def device_state_attributes(self):
         if self.data.has_powermeter and \
            self.data.current_consumption != STATE_UNKNOWN and \
            self.data.total_consumption != STATE_UNKNOWN:
-            attrs[ATTR_CURRENT_CONSUMPTION] = "%.1f %s" % \
-                (self.data.current_consumption, ATTR_CURRENT_CONSUMPTION_UNIT)
-            attrs[ATTR_TOTAL_CONSUMPTION] = "%.3f %s" % \
-                (self.data.total_consumption, ATTR_TOTAL_CONSUMPTION_UNIT)
+            attrs[ATTR_CURRENT_CONSUMPTION] = "%.1f" % \
```

Comment:
It would also be nice if you switch to the "[new](https://docs.python.org/3.6/library/string.html#format-string-syntax)" formatting syntax. Thanks for the extra effort.